### ISSUE TITLE
"Logger specified in config cannot be loaded" if system is slow to start

### ISSUE THREAD

**Issue by mkb137** [akkadotnet/akka.net#3130]
Version: GitHub as of 2017-09-27.  Latest is 1.3.1.
Environment: WCF service, VS 2017, Windows 10

In unit tests and in a WPF app, Akka starts fine but in a WCF service I'm getting "Logger [{logger name}] specified in config cannot be loaded.".  This is not a problem with the logger but a problem with the system attempting to send the logger a message before the system has initialized.

I traced the error message to `LoggingBus.AddLogger`.  The logger class is created correctly but when it is sent the `InitializeLogger` message the system hasn't yet started and so the logger is still an `UnstartedCell`.  The logger fails to respond to the message, the line `response = askTask.Result` throws an exception and we get the message that the logger could not be loaded.

When the system works, the system actor and its guardians are created before `LoggingBus.StartDefaultLoggers` is ever called.  
When it fails, `LoggingBus.StartDefaultLoggers` is executing before the  actor system is actually up and running.  I imagine that this is just a function of threads being slow to be created or something.  In any case, if adding and initializing a logger depends on the actor system or its guardians to be initialized then there should be some initial check or gate.

**Comment by mkb137** [comment]
I'll note for anyone with the same issue that my workaround is to remove all loggers, e.g:
```
akka {
	loggers = []
}
```

**Comment by Aaronontheweb** [comment]
I've been able to confirm and reproduce this bug here: https://github.com/petabridge/NBench/issues/314

The reason why I've been able to reproduce that issue is rather interesting - it comes from this code inside the `NBenchRunner` here: https://github.com/petabridge/NBench/blob/d83b040ea5a28e0ece6a3b0350b7b0d82bba9a28/src/NBench/Sdk/TestRunner.cs#L70-L95

When we set the processor affinity:

```csharp
 var proc = Process.GetCurrentProcess();
 proc.ProcessorAffinity = new IntPtr(2); // strictly the second processor!
```

This limits the concurrency down to a single logical CPU, so everything in the Akka.NET `ActorSystem` startup run from this benchmark has to execute in a single-threaded fashion. The default logger times out here:

https://github.com/akkadotnet/akka.net/blob/6293948ef086b0ed58ddf1ebc4c0a02291ae1f89/src/core/Akka/Event/LoggingBus.cs#L166-L191

the `Task.Result` call on the `Ask` operation is where we're vulnerable to timing out. This code should be rewritten so the loggers can start asynchronously. Might mean having to stick some `await` calls in there and propagating `Task` calls all the way up to `ActorSystem.Create` - might be some other ways of doing it that are less invasive though.
